### PR TITLE
OSX: Compress DMG

### PR DESCRIPTION
**Pull request by jflyper**
UDB0 is good enough.

### CODE REVIEW

File: gulpfile.js [hunk 1]
```diff
@@ -322,7 +322,8 @@ function release_osx64() {
                     { 'x': 448, 'y': 342, 'type': 'link', 'path': '/Applications' },
                     { 'x': 192, 'y': 344, 'type': 'file', 'path': pkg.name + '.app', 'name': 'Betaflight Configurator.app' }
                 ]
-            }
+            },
+            format: 'UDB0'
```
Comment (mikeller) [inline gulpfile.js]: Weird, according to https://github.com/LinusU/node-appdmg#specification it looks like `format` should go inside `specification` (and the same applies to `background`, maybe that's why the background is not working, @basdelfos).

Comment (basdelfos) [reply gulpfile.js]: Doh, that's it. Now I remember that the background was working in the beginning when I used a separate json file.

Comment (basdelfos) [reply gulpfile.js]: @jflyper can you replace this from line 114:

```
    return gulp.src([])
        .pipe(appdmg({
            target: path.join(releaseDir, get_release_filename('macOS', 'dmg')),
            basepath: path.join(appsDir, pkg.name, 'osx64'),
            specification: {
                title: 'Betaflight Configurator',
                contents: [
                    { 'x': 448, 'y': 342, 'type': 'link', 'path': '/Applications' },
                    { 'x': 192, 'y': 344, 'type': 'file', 'path': pkg.name + '.app', 'name': 'Betaflight Configurator.app' }
                ],
                background: path.join(__dirname, 'images/dmg-background.png'),
                format: 'UDBZ',
                window: {
                    size: {
                        width: 638,
                        height: 479
                    }
                }
            },
        })
    );
```
It fixes the compression to move to `specification` and the correct enum value. Also fixes the background not showing in the dmg.

Comment (jflyper) [reply gulpfile.js]: Strange. The dmg file is 205MB, and still no background image...

Comment (jflyper) [reply gulpfile.js]: Tags should be double-quoted? Not mandatory???

Comment (basdelfos) [reply gulpfile.js]: Quoting is not mandatory as long as the name does not have special characters like spaces, dashes, ...

**Pull request merged**